### ISSUE TITLE
Warn user if connection url is invalid

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1994]
Users might accidentally put two `?` in the query string instead of using `&` for the second param. We should warn them if this happens and give a helpful error message.

(Example: https://prisma.slack.com/archives/CKQTGR6T0/p1585492612186300)

**Comment by janpio** [comment]
Related: https://github.com/prisma/prisma2/issues/2000

**Comment by mrmodise** [comment]
@janpio I will take this. Thanks

**Comment by anishagg17** [comment]
@janpio is this also closed?

**Comment by janpio** [comment]
No, this seems a reasonable small DX feature. But as it does not happen often, we will not be able to help figure out where to do things.

**Comment by maddhruv** [comment]
I would suggest if we can add a validation for database connection URLs. While generate/migrate it would run validations before even trying to connect to the DB

**Comment by janpio** [comment]
Exactly. In this case it would be a special case to detect two invalid `?` in the connection url query parameters, that would allow us to give a _very_ specific error message with concrete steps what to fix.

**Comment by maddhruv** [comment]
Hey @janpio what do you propose? Adding a validation for all kinds of urls? or just checking for this certain case that you have mentioned?
Correct me if there already exists a validation for connection URIs

**Comment by janpio** [comment]
I think we can do both: Warn about invalid connection string URLs and warn about this special case that we understand really well and that is a common case unexperienced users encounter.

**Comment by 123om123** [comment]
is anyone working on this?

**Comment by rv602** [comment]
@janpio is this issue still open? I would like to work on it.
